src/components/ChemicalEditors.vue

```diff
@@ -40,11 +32,26 @@ export default {
   },
   data() {
     return {
-      molfileChanged: false,
-      mrvfileChanged: false
+      editorChanged: false
     };
```

Comment:
good point, I've tracked changes in either editor to inform the dis/ability of the button and currently it will respond between flipping to either ketcher or marvin independently

Reply:
Functionality looks good.